sdks/python/apache_beam/utils/timestamp.py

```diff
@@ -57,6 +60,7 @@ def __init__(self, seconds=0, micros=0):
 
   @staticmethod
   def of(seconds):
+    # type: (Union[int, float, Timestamp]) -> Timestamp
```

Comment:
Somewhere in this file would be a good place to define this Union type.

Reply:
done